modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/IgniteAbstractTransactionImpl.java

```diff
@@ -47,9 +47,7 @@ public abstract class IgniteAbstractTransactionImpl implements InternalTransacti
     /** The tracker is used to track an observable timestamp. */
     protected final HybridTimestampTracker observableTsTracker;
 
-    /**
-     * Transaction coordinator inconsistent ID.
-     */
+    /** Transaction coordinator inconsistent ID. */
```

Comment:
```suggestion
    /** Transaction coordinator ephemeral ID. */
```
'Inconsistent' sounds like something causes troubles, and here it's just about something that can change. Although, if we change it, we might want to do it *consistently* (pun intended), not just here, so if it's used in more than a couple of places, maybe it's better to file a separate issue.